shard/src/main/java/com/redhat/developer/shard/utils/WebClientUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.redhat.developer.shard.utils;
+
+import java.net.URI;
+
+import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.ext.web.client.WebClient;
+
+public class WebClientUtils {
+
+    public static WebClient getClient(Vertx vertx, String url) {
```

Comment:
Does Quarkus offer us an out-of-the-box way to configure the `WebClient`?

Reply:
Actually it is not quarkus but vertx. Anyway nope afaik